pages/login.tsx

```diff
@@ -7,6 +7,8 @@ import { LoginWithGooglePopUp, auth } from "@/firebase";
 import { useAuthState } from "react-firebase-hooks/auth";
 
 import { DarkModeToggle } from "../Components/components";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
```

Comment:
I think this should not be imported here. It should go inside `_app.tsx`. As we will be using this `toast` throughout the whole application so importing it again and again will not be the right approach.